src/server/report/result.js

```diff
@@ -189,6 +187,6 @@ export function renderBrowserResults(browser, tests, options) {
 				<div>version ${browser.version}</div>
 			</div>
 		</div>
-		${results}
+		${filteredTests.length === 0 ? renderEmpty() : results}
```

Comment:
Since a browser's results can now be empty, one more place for sad magnifying glass!

<img width="630" alt="Screenshot 2023-07-19 at 4 02 00 PM" src="https://github.com/BrightspaceUI/testing/assets/5491151/42ea6ca7-310b-44f5-bedc-346bb73c78e8">